src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -149,6 +150,8 @@ public async Task PostExport()
                 {
                     HttpContext.Abort();
                 }
+                
+                LogTaskToAudit("exprot-data", operationId, configuration: null);
```

Comment:
```suggestion
                LogTaskToAudit("export-data", operationId, configuration: null);
```

can we use `Operations.OperationType.DatabaseExport` instead of `"export-data"`?